Automattic/newspack-popups
# Add above header placement option

### All Submissions:






### Changes proposed in this Pull Request:



Adds an "above header" campaign placement, as requested in #260

Additional changes in Theme (https://github.com/Automattic/newspack-theme/pull/1131) and the plugin (https://github.com/Automattic/newspack-plugin/pull/691)

### How to test the changes in this Pull Request:

1. Create a new campaign, observe a new option in "Placement" select – "Above site header"
2. Use this option
3. Verify that the campaign is placed above site header
4. Filtering (category, tag, etc.) and everything else should work as with a regular inline campaign

### Other information:
